server.js

```diff
@@ -0,0 +1,54 @@
+import express from "express";
+import process  from "os";
+
+const app = express();
+app.use(express.json());
+
+const events =[];
+
+function isValidUTCDate(dateString){
+  const date = new Date(dateString);
+   if(date.toISOString() === dateString){
+    return true;
+   }
+
+  return false;
+}
+
+app.get("/health-check", (req,res)=> {
+  const healthCheck = {
+    uptime: process.uptime(),
+    message: "OK",
+    timestamp: Date.now(),
+  };
+
+  try{
+    res.send(healthCheck);
+  } catch(error){
+    healthCheck.message= error
+    res.status(500).send()
+  }
```

Comment:
```suggestion
  } catch(error){
    healthCheck.message= error
    res.status(500).send()
  }
  } catch (error) {
    console.error("Error sending health check:", error);
    res.status(500).send({
      message: "Erro ao processar a solicitação de verificação de saúde. Tente novamente mais tarde."
    });
  }
```
1. Criar  Log de erro para facilitar a depuração
2. "healthCheck.message= error" dentro de **catch** não é usado pra nada
3. Retornar uma mensagem de erro que sejam úteis. Informar que ocorreu um erro mas não expor informações  sensíveis sobre a aplicação.